### PR TITLE
Release 1.0 - RC1

### PR DESCRIPTION
**Pull request by lszymik**
**Description**

Changes proposed in this pull request:

- Release 1.0

### CODE REVIEW

File: installation/resources/installer.yaml [hunk 1]
```diff
@@ -233,7 +233,7 @@ spec:
       serviceAccountName: kyma-installer
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:0fdc80dd
+        image: eu.gcr.io/kyma-project/installer:1.0.0-rc1
```
Comment (crabtree) [inline installation/resources/installer.yaml]:
```suggestion
        image: eu.gcr.io/kyma-project/kyma-installer:1.0.0-rc1
```

File: installation/resources/installer.yaml [hunk 2]
```diff
@@ -233,7 +233,7 @@ spec:
       serviceAccountName: kyma-installer
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:0fdc80dd
+        image: eu.gcr.io/kyma-project/kyma-installer:1.0.0-rc1
```
Comment (aszecowka) [inline installation/resources/installer.yaml]: installer and kyma-installer are 2 different images, please talk to @Tomasz-Smelcerz-SAP

Comment (crabtree) [reply installation/resources/installer.yaml]: We are aware of this, now there should be kyma-installer as this is also release artifact.

**Pull request merged**